### DISCUSSION TITLE
startx not installed with securityonion-2.3.250-20230519.iso

### DISCUSSION THREAD

**Discussion by NetwarSystem** [Security-Onion-Solutions/securityonion#10588] [Unsupported Versions] [answered]
I'm installing the analyst workstation under VirtualBox and it's reporting that startx is not found.

This is referenced on line 118 of /home/admin/SecurityOnion/setup/so-setup

I changed to / and did find . > find.txt

There's no sign of startx anywhere in there. There are just three X11 rpms and a little bit of stuff in /etc/X11. Searching for "x11" reveals more rpms, but nothing that looks like a correct install with /bin/, /lib/, etc.

I'm not sure what else I should report to assist with troubleshooting this.

**Comment by dougburks** [comment, marked as answer]
Does [this issue](https://github.com/Security-Onion-Solutions/securityonion/issues/8088) match what you're seeing?

If so, please try the Analyst Workstation option on the ISO boot menu:
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/1659467/71b8d468-dfaf-47dc-9bfb-45a16cc10f0a)

**Reply by NetwarSystem** [reply]
Why yes, there **IS** a distinct possibility that I am a confused old man yelling at clouds over here ...
